### DISCUSSION TITLE
Feature: Don't create "Battery last replaced" sensor and "Battery replaced" button for "Rechargeable" devices

### DISCUSSION THREAD

**Discussion by JeffersonBledsoe** [andrew-codechimp/HA-Battery-Notes#1021] [Ideas]
While you can replace the internal battery for some of these devices, I don't think the semantics are the same for replacing a battery once the device reaches 0% and replacing the battery because it no longer functions correctly

**Comment by andrew-codechimp** [comment]
I think the best way of handling this is to go into the device and disable/hide the entities.
If I try and make this automatic it will mean dealing with messy string comparison or a whole new flag in the library which for a few devices will be a lot of effort.

**Comment by JeffersonBledsoe** [comment]
Yeah, disabling the entities is what I'm currently doing, just figured it makes the most sense from a 'defaults' perspective. Appreciate the maintanence burden on implementing this though, I'll continue doing it manually! Thanks for the quick response and great integration 🙂
